api/v1/infinispan_types.go

```diff
@@ -283,6 +283,7 @@ const (
 )
 
 type InfinispanLoggingSpec struct {
+	Pattern    string                      `json:"pattern"`
```

Comment:
We need `omitempty` so the `Pattern` element is only included in the resource output if not `""`

Reply:
Please add the `// +optional` comment to this field.